### PR TITLE
[xdoctest] reformat example code with google style in No.307

### PR DESCRIPTION
**Pull request by Liyulingyue**
### PR types

Others 
### PR changes

Others 
### Description

https://github.com/PaddlePaddle/Paddle/issues/55629

### CODE REVIEW

File: python/paddle/distributed/entry_attr.py [hunk 1]
```diff
@@ -22,22 +22,22 @@ class EntryAttr:
...
-                entry=entry,
-                param_attr=paddle.ParamAttr(name="SparseFeatFactors",
-                                           initializer=paddle.nn.initializer.Uniform()))
+            >>> emb = paddle.static.nn.sparse_embedding((
+            ...     input=input,
+            ...     size=[sparse_feature_dim, embedding_size],
+            ...     is_test=False,
+            ...     entry=entry,
+            ...     param_attr=paddle.ParamAttr(name="SparseFeatFactors",
+            ...                                initializer=paddle.nn.initializer.Uniform()))
```
Comment (SigureMo) [inline python/paddle/distributed/entry_attr.py]:
```suggestion
            >>> emb = paddle.static.nn.sparse_embedding(
            ...     input=input,
            ...     size=[sparse_feature_dim, embedding_size],
            ...     is_test=False,
            ...     entry=entry,
            ...     param_attr=paddle.ParamAttr(
            ...         name="SparseFeatFactors",
            ...         initializer=paddle.nn.initializer.Uniform()
            ...     )
            ... )
```

File: python/paddle/distributed/entry_attr.py [hunk 2]
```diff
@@ -59,22 +59,22 @@ class ProbabilityEntry(EntryAttr):
...
```
Comment (SigureMo) [inline python/paddle/distributed/entry_attr.py]:
```suggestion
            >>> emb = paddle.static.nn.sparse_embedding(
            ...     input=input,
            ...     size=[sparse_feature_dim, embedding_size],
            ...     is_test=False,
            ...     entry=entry,
            ...     param_attr=paddle.ParamAttr(
            ...         name="SparseFeatFactors",
            ...         initializer=paddle.nn.initializer.Uniform()
            ...     )
            ... )
```

File: python/paddle/distributed/entry_attr.py [hunk 3]
```diff
@@ -100,22 +100,22 @@ class CountFilterEntry(EntryAttr):
...
```
Comment (SigureMo) [inline python/paddle/distributed/entry_attr.py]:
```suggestion
            >>> emb = paddle.static.nn.sparse_embedding(
            ...     input=input,
            ...     size=[sparse_feature_dim, embedding_size],
            ...     is_test=False,
            ...     entry=entry,
            ...     param_attr=paddle.ParamAttr(
            ...         name="SparseFeatFactors",
            ...         initializer=paddle.nn.initializer.Uniform()
            ...     )
            ... )
```

File: python/paddle/distributed/entry_attr.py [hunk 4]
```diff
@@ -144,25 +144,25 @@ class ShowClickEntry(EntryAttr):
...
-                entry=entry,
-                param_attr=paddle.ParamAttr(name="SparseFeatFactors",
-                                           initializer=paddle.nn.initializer.Uniform()))
+            >>> emb = paddle.static.nn.sparse_embedding(
+            ...     input=input,
+            ...     size=[sparse_feature_dim, embedding_size],
+            ...     is_test=False,
+            ...     entry=entry,
+            ...     param_attr=paddle.ParamAttr(name="SparseFeatFactors",
+            ...                                initializer=paddle.nn.initializer.Uniform()))
```
Comment (SigureMo) [inline python/paddle/distributed/entry_attr.py]:
```suggestion
            >>> emb = paddle.static.nn.sparse_embedding(
            ...     input=input,
            ...     size=[sparse_feature_dim, embedding_size],
            ...     is_test=False,
            ...     entry=entry,
            ...     param_attr=paddle.ParamAttr(
            ...         name="SparseFeatFactors",
            ...         initializer=paddle.nn.initializer.Uniform()
            ...     )
            ... )
```

File: python/paddle/distributed/entry_attr.py [hunk 5]
```diff
@@ -22,22 +22,25 @@ class EntryAttr:
     Examples:
         .. code-block:: python
 
-            import paddle
+            >>> import paddle
```
Comment (Liyulingyue) [inline python/paddle/distributed/entry_attr.py:25]:
```suggestion
            >>> import paddle
            >>> paddle.enable_static()
```

File: python/paddle/distributed/entry_attr.py [hunk 6]
```diff
@@ -59,22 +62,25 @@ class ProbabilityEntry(EntryAttr):
...
```
Comment (Liyulingyue) [inline python/paddle/distributed/entry_attr.py:66]:
```suggestion
            >>> import paddle
            >>> paddle.enable_static()
```

File: python/paddle/distributed/entry_attr.py [hunk 7]
```diff
@@ -100,22 +106,25 @@ class CountFilterEntry(EntryAttr):
...
```
Comment (Liyulingyue) [inline python/paddle/distributed/entry_attr.py:111]:
```suggestion
            >>> import paddle
            >>> paddle.enable_static()
```

**Review by SigureMo — APPROVED** [review]
ฅ LGTMeow ฅ

**Pull request merged**